doc/etc/rose-rug-advanced-tutorials-macro/planet.py.report.html

```diff
@@ -0,0 +1,125 @@
+  "../../css/rose-doc.css" />
+  <link rel="stylesheet" type="text/css" href=
+  "../../google-code-prettify/prettify.css" />
+  <script type="text/javascript" src="../../js/jquery.min.js">
+</script>
+  <script type="text/javascript" src="../../js/run_prettify.js">
+</script>
+  <script type="text/javascript" src=
+  "../../js/prettify-rose-conf.js">
+</script>
+  <script type="text/javascript" src=
+  "../../js/prettify-cylc-suite-rc.js">
+</script>
+  <script type="text/javascript" src="../../js/bootstrap.min.js">
+</script>
+  <script type="text/javascript" src="../../js/rose-doc.js">
+</script>
+  <script type="text/javascript" src="../../js/rose-version.js">
+</script>
+</head>
+
+<body>
+  <nav class="navbar navbar-default">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="."><span class="logo">Rose
+        Documentation</span></a>
+      </div>
+      <!-- Collect the nav links, forms, and other content for toggling -->
+
+      <div class="collapse navbar-collapse" id=
+      "bs-example-navbar-collapse-1">
+        <ul class="nav navbar-nav navbar-right">
+          <li><span class="navbar-text"><span class=
+          "compliance">&#169; British Crown Copyright 2012-6
+          <a href="http://www.metoffice.gov.uk">Met Office</a>. See
+          <a href="rose-terms-of-use.html">Terms of Use</a>.<br />
+          This document is released under the <a href=
+          "http://www.nationalarchives.gov.uk/doc/open-government-licence/"
+          rel="license">Open Government
+          Licence</a>.</span></span></li>
+
+          <li><span id="rose-version" class=
+          "navbar-text"></span></li>
+        </ul>
+      </div><!-- /.navbar-collapse -->
+    </div><!-- /.container-fluid -->
+  </nav>
+
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <div class="panel panel-default">
+          <div class="panel-body">
+            <h2>planet.py</h2>
+            <pre class="prettyprint">
+class PlanetReporter(rose.macro.MacroBase):
+
+    """Creates a report on the value of env=WORLD."""
+
+    GENERIC_HOROSCOPE_STATEMENTS = [
+        'be cautious', 'remain indoors', 'expect the unexpected',
+        'not walk under ladders', 'seek new opportunities']
+
+    def report(self, config, meta_config=None):
+        world_node = config.get(["env", "WORLD"])
+        if world_node is None or world_node.is_ignored():
+            return
+        planet = world_node.value
+        if planet.lower() == 'earth':
+            print 'Please choose a planet other than Earth.'
+            return
+        constellation = self.get_planet_info(planet)
+        if not constellation:
+            print 'Could not find horoscope entry for {0}'.format(planet)
+            return
+        else:
+            print (
+                '{planet} is currently passing through {constellation}.\n'
+                'You should {tosh} today.'
```

Comment:
Can you pick something other than "tosh" e.g. "statement" or "advice". If nothing else it's colloquial...